manifests/role/app_host/dev.pp

```diff
@@ -17,4 +17,5 @@
   include nebula::profile::ruby
   include nebula::profile::nodejs
   include nebula::profile::named_instances
+  include nebula::profile::yarn
```

Comment:
Along the same line, we want this (and it's already installed) on nectar, but do we also want it on punch, champagne, and search-prep?